third_party/xla_client/xrt_computation_client.h

```diff
@@ -252,15 +252,15 @@ class XrtComputationClient : public ComputationClient {
   std::vector<DataPtr> TransferToServer(
       absl::Span<const TensorSource> tensors) override;
 
-  void TransferToServer(absl::Span<const TensorSource> tensors,
-                        absl::Span<const DataPtr> datas) override;
-
   DataPtr TransferShardsToServer(absl::Span<const TensorSource> tensor_shards,
                                  std::string device,
                                  xla::Shape shape) override {
     XLA_ERROR() << __FUNCTION__ << " not implemented";
   }
 
+  void TransferToServer(absl::Span<const TensorSource> tensors,
```

Comment:
why the move?

Reply:
To put them in the same order as defined in the parent class. Come to think of it, it would look cleaner if we re-order the parent methods instead.